sources/lib/ParameterJuicer.php

```diff
@@ -281,7 +287,7 @@ private function setDefaultValues(array $values): array
     {
         foreach ($this->default_values as $field => $default_value) {
             if (!isset($values[$field]) && !array_key_exists($field, $values)) {
-                $values[$field] = $default_value;
+                $values[$field] = $default_value();
```

Comment:
I don't know but since php 5.4.0 you can call any callable stored in a variable and the documentation page refer to the `call_user_func` documentation pqge http://php.net/manual/en/functions.variable-functions.php

I wrote like this because it's an habit. I use lot of invokable classes and I use them like this.

But in all the code you use `call_user_func` so it is maybe a good think to use it everywhere.